### ISSUE TITLE
Coke Oven Animations on the Light on and Off don't always seem to sync correctly. 

### ISSUE THREAD

**Issue by KaneHart** [GTNewHorizons/GT-New-Horizons-Modpack#8041]
#### Which modpack version are you using?
2.1.0.7a
#
#### If in multiplayer; On which server does this happen? 
http://stonelegion.com 
#
#### What did you try to do, and what did you expect to happen?
Put Logs in and Watch it turn on visually or when running out to turn off visually. 
#
#### What happened instead? (Attach screenshots if needed)
Sometimes it would not turn on or off visually. Sort of like it was not sending packets correctly on the status of the block?
#
#### What do you suggest instead/what changes do you propose?

#
#### What is your GTNH Discord username?
Kane Hart#6969

**Comment by Dream-Master** [comment]
I guess it is because of the ressourcepack "Realistic Sky GT New Horizons" in the ressourcepack folder

**Comment by KaneHart** [comment]
Yes I have that enabled. Is it something that can be fixed or just an issue with the way textures are done in it? Does it effects others visuals also? 

I think others said they saw it too, but I could be wrong I will follow up tomorrow and ask people.

**Comment by Methes** [comment]
This wasn't the case before (and I've always used the Realistick Sky resource pack).
I think I've noticed it since the ovens started producing pollution.
You can't rely on the texture at all now.

**Comment by Prometheus0000** [comment]
What leads you to think it's the resourcepack? Also, did you try without the resourcepack?

**Comment by KaneHart** [comment]
https://cdn.discordapp.com/attachments/401118216228831252/853309104112795679/2021-06-12_12-24-06.mp4

With all resources off. Good example of it acting up..

**Comment by Prometheus0000** [comment]
Maybe it's one of the mixin/ASMs? Try disabling/removing those. Hodgepodge, modmixins, SpongeMixins are the ones I remember, there might be another?

> I think I've noticed it since the ovens started producing pollution.

It could be because the thing to add pollution to coke ovens messed things up.

**Comment by Prometheus0000** [comment]
```
# Adds Pollution to the Railcraft Boilers, Machines, etc.
    B:fixRailcraftPollution=true
```

In the mod mixins config. Try disabling it if removing the mod helps.
